Gio-2.0.gir

```diff
@@ -4230,6 +4246,19 @@ can cancel the startup notification.</doc>
           </parameter>
         </parameters>
       </glib:signal>
+      <glib:signal name="launch-started" when="last">
```

Comment:
This is weird, it definitely has a since annotation upstream but somehow it's not part of the gir file